deploy/kubernetes/helm/che/templates/configmap.yaml

```diff
@@ -40,9 +40,19 @@ data:
   CHE_INFRA_KUBERNETES_TLS__SECRET: ""
 {{- end }}
 {{- if .Values.global.multiuser }}
-  CHE_KEYCLOAK_AUTH__SERVER__URL: {{ template "keycloakAuthUrl" . }}
   CHE_KEYCLOAK_CLIENT__ID: {{ .Values.cheKeycloakClientId }}
+{{- end }}
+{{- if .Values.global.multiuser }}
+  CHE_KEYCLOAK_AUTH__SERVER__URL: {{ template "keycloakAuthUrl" . }}
```

Comment:
Can you elaborate why did you place setting of `CHE_KEYCLOAK_CLIENT__ID` and `CHE_KEYCLOAK_AUTH__SERVER__URL` into two separated sections with the same condition?
Couldn't they be in the same like:
```
{{- if .Values.global.multiuser }}
  CHE_KEYCLOAK_CLIENT__ID: {{ .Values.cheKeycloakClientId }}
  CHE_KEYCLOAK_AUTH__SERVER__URL: {{ template "keycloakAuthUrl" . }}
  CHE_KEYCLOAK_REALM: {{ .Values.cheKeycloakRealm }}
{{- end }}
```

Reply:
my mistake, i initially wanted to separate 
CHE_KEYCLOAK_AUTH__SERVER__URL, CHE_KEYCLOAK_REALM into separate condition 
(multiuser and oidc is not used)
but in case oidc is defined they are ignored anyway, so there is no need to overcomplicate conditions